src/math/rem_pio2_large.rs

```diff
@@ -0,0 +1,489 @@
+use super::scalbn;
+use super::floor;
+
+/// double x[],y[]; int e0,nx,prec;
```

Comment:
these can be turned into plain comments (`//` instead of `///`). Doc comments won't be rendered for this function / const / item because it's private.